src/utils.tsx

```diff
@@ -76,3 +76,14 @@ export function getText(
     return type;
   }
 }
+
+export const debounce = <T extends (...args: unknown[]) => unknown>(
+  callback: T,
+  delay = 250,
+): ((...args: Parameters<T>) => void) => {
+  let timeoutId: NodeJS.Timeout
```

Comment:
I don't think this is a NodeJS timeout. It's probably just number.

Can use window.setTimeout instead to get the correct type I think.

Reply:
Ohh yeah , should use window.setTimeout one